config/urls.py

```diff
@@ -7,13 +7,16 @@
 from poznaj.points.views import PointsViewSet
 from poznaj.stories.views import StoriesViewSet
 
+from poznaj.views import MainView
+
 router = routers.DefaultRouter()
 router.register(r'images', ImagesViewSet)
 router.register(r'points', PointsViewSet)
 router.register(r'stories', StoriesViewSet)
 
 urlpatterns = [
     url(settings.ADMIN_URL, admin.site.urls),
+    url(r'^', MainView.as_view()),
```

Comment:
Please change it to `^$` as tests are broken

Reply:
👍  ok